taipy/core/setup.py

```diff
@@ -45,6 +45,9 @@
     "mssql": ["pyodbc>=4,<4.1"],
     "mysql": ["pymysql>1,<1.1"],
     "postgresql": ["psycopg2>2.9,<2.10"],
+    "parquet": ["fastparquet==2022.11.0"],
+    "s3": ["boto3==1.29.1"],
+    "mongo": ["pymongo[srv]>=4.2.0,<5.0"],
```

Comment:
We need to remove the `"pymongo[srv]>=4.2.0,<5.0",` from the requirements right?